addons/pos_restaurant/static/tests/tours/helpers/ProductScreenTourMethods.js

```diff
@@ -52,9 +52,41 @@ class DoExt extends Do {
             },
         ];
     }
+    clickOrderButton() {
+        return [
+            {
+                content: "click order button",
+                trigger: ".actionpad .submit-order",
+            },
+        ];
+    }
 }
 
 class CheckExt extends Check {
+    orderlinesIsNotToOrder() {
+        return [
+            {
+                content: "click order button",
+                trigger: ".actionpad .submit-order:not(.highlight)",
```

Comment:
Since this step should just be a check, put `run() {}`.

Reply:
We did not need it anymore. This utility function is removed with the previous split.